source/_docs/code-of-conduct.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Pantheon Community Code of Conduct
+description: Pantheon is dedicated to a positive and harassment-free community experience for everyone.
+categories: [managing]
+tags: [community]
+contributors:
+  - joshkoenig, rachelwhitton
+date: 9/27/2016
+---
+
+# Contributor Covenant Code of Conduct
+
```

Comment:
Suggest deleting H1 or using H2 instead, [seems a little out of wack as is](https://codeofcon-static-docs.pantheonsite.io/docs/code-of-conduct): 
<img width="1393" alt="screen shot 2016-09-27 at 5 05 22 pm" src="https://cloud.githubusercontent.com/assets/10119525/18893699/a5908a58-84d4-11e6-88d4-a252845f118f.png">